### PR TITLE
tractorgen: add test

### PR DESCRIPTION
**Pull request by Moisan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Add a test to `tractorgen`. Related to #11898 .

### CODE REVIEW

File: Formula/tractorgen.rb [hunk 1]
```diff
@@ -18,4 +18,16 @@ def install
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    expected = <<-'EOS'
```
Comment (ilovezfs) [inline Formula/tractorgen.rb]: Please use `<<~'EOS'` in case we start auditing `<<-`.

Comment (Moisan) [reply Formula/tractorgen.rb]: <<~'EOS' seems to strip whitespaces in the string when all lines start with whitespaces, as is the case here. Am I missing something?

Comment (ilovezfs) [reply Formula/tractorgen.rb]: Hmm... I guess we'll just have to figure out how to deal with this one later if we do add such an audit.

**Comment by ilovezfs** [comment]
Thanks again @Moisan! :tractor:

**Pull request merged**